modules/restore-hosted-cluster-virt.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assembly:
+//
+// * hosted_control_planes/hcp_high_availability/hcp-backup-restore-virt.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="restore-hosted-cluster-virt_{context}"]
+= Restoring a hosted cluster on {VirtProductName}
+
+After you back up a hosted cluster on {VirtProductName}, you can restore the backup.
+
+[NOTE]
+====
+The restore process can be completed only on the same management cluster where you created the backup.
+====
+
+.Procedure
+
+. Ensure that no pods or persistent volume claims (PVCs) are running in the `HostedControlPlane` namespace.
+
+. Delete the following objects from the management cluster:
+
+** `HostedCluster`
+** `NodePool`
+** PVCs
+
+. Create a YAML file to serve as the restoration manifest that is similar to the following example:
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: hc-clusters-hosted-restore
+  namespace: openshift-adp
+spec:
+  backupName: hc-clusters-hosted-backup
+  restorePVs: true <1>
+  existingResourcePolicy: update <2>
+  excludedResources:
+  - nodes
+  - events
+  - events.events.k8s.io
+  - backups.velero.io
+  - restores.velero.io
+  - resticrepositories.velero.io
+----
++
+<1> This field starts the recovery of pods with the included persistent volumes.
+<2> By setting `existingResourcePolicy` to `update` ensures that any objects are overwritten with backup content. This action can cause issues with objects that contain immutable fields, which is why you deleted the `HostedCluster`, node pools, and PVCs. If you do not set this policy, the Velero engine skips the restoration of objects that already exist.
```

Comment:
```suggestion
<2> Setting `existingResourcePolicy` to `update` ensures that any existing objects are overwritten with backup content. This action can cause issues with objects that contain immutable fields, which is why you deleted the `HostedCluster`, node pools, and PVCs. If you do not set this policy, the Velero engine skips the restoration of objects that already exist.
```

Reply:
Since the option of skipping this is mentioned, are there any consequences or considerations that should be covered? If you don't set the policy, did you need to delete the objects at the start?

Reply:
@nunnatsa Jeana has another good question here. Can you answer it?